templates/crate/details.html

```diff
@@ -69,7 +69,7 @@
                                     {# If the repo link is for github or gitlab, show some stats #}
                                     {# TODO: add support for hosts besides github and gitlab (#35) #}
                                     {%- if let Some(repository_metadata) = details.repository_metadata -%}
-                                        {{ "code-branch"|fab(false, false, "") }}
+                                        {{ "code-branch"|fab(false, false, "")|safe }}
```

Comment:
@GuillaumeGomez just a thought, 

from what I see, we _always_ need `|safe` when using `fab`. So why can't `fab` itself directly add it?

Reply:
It's already implemented in `rinja` (the possibility to mark a filter "safe"), but not yet released. The next release of rinja should be in the next days. It also adds `is (not) defined` which we used a lot before. So once the release is done, I'm planning to do a lot of cleanups.